website/versioned_docs/version-2.0.0-beta.14/advanced/routing.md

```diff
@@ -0,0 +1,17 @@
+# Routing
```

Comment:
not sure to understand the purprose of this file, is it just temporary until the next releases?

Reply:
Yes, because I want to allow readers to switch between versions and understand how the same page looks like across versions....